_posts/2025-02-20-use-desctop-visual-studio-code.adoc

```diff
@@ -0,0 +1,108 @@
+---
+title: "Develop in local Visual Studio Code using Eclipse Che as a remote Cloud Development Environment"
+layout: post
+author: Vitaliy Gulyy
+description: >-
+   Use your local Visual Studio Code as an editor for a Che workspace
+categories: []
+keywords: ['Visual Studio Code', 'Eclipse Che editor']
+slug: /@vgulyy/use-desktop-visual-studio-code
+---
+
+== Introduction
+
+Eclipse Che provides developers a web version of the of link:https://github.com/microsoft/vscode[Visual Studio Code OSS] editor, that includes all the features of the desktop version. But for developers, the transition to cloud development can be quite unusual.
+With link:https://code.visualstudio.com/docs/editor/command-line[Visual Studio Code Command Line Interface (CLI)] you can create a tunnel that alows you connect your familiar local Visual Studio Code to your workspace.
+
+== Connecting Visual Studio Code desktop to a running workspace
```

Comment:
I propose to swap places and start with `Connecting Visual Studio Code desktop to a new workspace`

Reply:
The order is changed